_posts/2020-01-07-aksengine-advanced-networking-part1.md

```diff
@@ -1,18 +1,9 @@
-# Understanding Networking Options in Azure AKS-Engine [Part 1]
-
-- [Understanding Networking Options in Azure AKS-Engine [Part 1]](#understanding-networking-options-in-azure-aks-engine-part-1)
```

Comment:
Oh, TOC was helpful, indeed. You didn't need to remove it.

Reply:
I had removed the TOC because it was being automatically added to the Blog home page (normally, the first paragraph is added). But, then I found a solution that creates a TOC and doesn't cause the same problem.

This solution just adds the following snippet before the very first paragraph and that's it.

```
* TOC
{:toc}
```

https://ouyi.github.io/post/2017/12/31/jekyll-table-of-contents.html

Reply:
@deepnetworkgmbh/developers, please note the solution @ahmddp found for TOC, it's neat.

Also, what do you think about @emrahkaya's question? I like the `asset/blogtitle` proposal to keep things clean. Though I think it should be the full title with the date. I cannot help but wonder if there is some plugin etc to help the developer with this.